External/AvalonTools/test1.cpp

```diff
@@ -144,14 +144,14 @@ void test3() {
     ExplicitBitVect bv(2048);
     AvalonTools::getAvalonFP("c1cocc1", true, bv, 2048, false, true, 0x006FFF);
     BOOST_LOG(rdInfoLog) << "c1cocc1 " << bv.getNumOnBits() << std::endl;
-    TEST_ASSERT(bv.getNumOnBits() == 53);
+    TEST_ASSERT(bv.getNumOnBits() == 48);
```

Comment:
I believe that all the changes in this file are not valid for v1.2 of the Avalon toolkit.